.gitignore

```diff
@@ -9,4 +9,5 @@ MANIFEST
 .coverage
 .idea/
 __pycache__
-netlib.egg-info/
\ No newline at end of file
+netlib.egg-info/
+pathod/
```

Comment:
This shouldn't be neccessary if you install mitmproxy using the instructions in https://github.com/mitmproxy/mitmproxy#hacking